spec/support/configurations_shared_context.rb

```diff
@@ -10,7 +10,11 @@ class User
       field :locked_at, type: Time
 
       field :age, type: Integer
+
+      index({ _id: 1, locking_name: 1 }, name: 'mongoid_locker_index', sparse: true, unique: true, expire_after_seconds: lock_timeout)
     end
+
+    User.create_indexes
```

Comment:
Depending on the `database_cleaner` strategy the collections may be dropped (I didn't check) between tests, not just across `:context`. I think we want to move the `User` class out in its own file loaded once, and the creation of indexes along with `reset!` into a `before do`.

Reply:
We didn't use `database_cleaner` gem here, if you meant it.
This `index` was added with only one purpose - check that it works before it is added into `README`. So, the index works and does not impact on anything.
We can apply `Mongoid.purge!` if it is really needed here.

Not sure that understood your idea properly.

Reply:
Yeah ignore me. I was having a conversation with myself in my head without looking at the whole code ;) All is good.